strr-api/src/strr_api/responses/RegistrationSerializer.py

```diff
@@ -392,6 +392,8 @@ def populate_host_registration_details(cls, registration_data: dict, registratio
             "jurisdiction": RegistrationSerializer.get_jurisdiction_from_application(registration),
             "prRequired": registration.rental_property.pr_required,
             "blRequired": registration.rental_property.bl_required,
+            "rentalUnitSetupOption": registration.rental_property.rental_space_option,
+            "hostType": registration.rental_property.host_type,
         }
```

Comment:
I think it is missing `strataHotelRegistrationNumber` here.

Reply:
It is already there